Makefile

```diff
@@ -188,15 +190,11 @@ test\:build:
 
 .PHONY: test\:ci
 test\:ci:
```

Comment:
todo: suggest this should not be named as `make test:ci` anymore as it doesn't make sense anymore. Perhaps something like `make coverage`. You can then reuse this make rule to replace the first 5 lines of `test\:coverage:` rule as well, it's now generating code coverage of all the tests (of course and running all tests too).

Reply:
I've merged the coverage tests like you suggested.